### PR TITLE
[JENKINS-47491] Ie11 fixes

### PR DESCRIPTION
**Pull request by scherler**
# Description

https://issues.jenkins-ci.org/browse/JENKINS-47491

https://github.com/PolymerElements/iron-iconset-svg/issues/46

Any SVG elements written by SvgIcon will be put into the tab order of the page in IE11.

# Expected outcome

SVG elements should not be focusable by default

# Submitter checklist
- [x] Link to JIRA ticket in description, if appropriate.
- [x] Change is code complete and matches issue description
- [x] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [x] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

### CODE REVIEW

**Comment by michaelneale** [comment]
ping @sophistifunk @NicuPascu can one of you review?

**Comment by sophistifunk** [comment]
LGTM. Wasn't this already done earlier? Or should I lay off the shrooms?

**Comment by michaelneale** [comment]
@sophistifunk

File: jenkins-design-language/src/js/components/material-ui/SvgIcon/SvgIcon.js [hunk 1]
```diff
@@ -66,6 +70,7 @@ class SvgIcon extends Component {
     return (
       <svg
         {...other}
+        focusable={focusable}
```
Comment (kzantow) [inline jenkins-design-language/src/js/components/material-ui/SvgIcon/SvgIcon.js]: Why not just drop the option and have `focusable={false}`? ... but wasn't this change already done? (as @sophistifunk said)

**Comment by scherler** [comment]
https://github.com/jenkinsci/blueocean-plugin/commit/06a1470c6110ec83094dd1fb8af30498421c49f3#diff-9d44d2134191673503efac16ed92b580 @sophistifunk yeah lots of places and dropped the prop

**Pull request merged**